from fastapi import FastAPI, HTTPException
import pandas as pd
import numpy as np


# cargamos los datos
